config/03-ui/31-ui-deployment.yaml

```diff
@@ -19,26 +19,26 @@ spec:
         - name: ui
           image: quay.io/tekton-hub/ui
           env:
-            - name: GHE_URL
+            - name: API_URL
               valueFrom:
                 configMapKeyRef:
                   name: ui
-                  key: GHE_URL
-            - name: API_URL
+                  key: API_URL
+            - name: API_VERSION
               valueFrom:
                 configMapKeyRef:
                   name: ui
-                  key: API_URL
-            - name: GH_CLIENT_ID
+                  key: API_VERSION
+            - name: AUTH_BASE_URL
```

Comment:
how does UI now access GH_CLIENT_ID ?
for GH
using an api?

Reply:
Since we have a new OAuth server, so everything is handled over there

Reply:
got it !